mainboards/Makefile

```diff
@@ -20,73 +20,8 @@ docker-build: Dockerfile
 
 docker-run-aeeon-up: docker-build
```

Comment:
If we still want docker build support for few boards, shouldn't we keep the Dockerfile?